node/modules/storageminer.go

```diff
@@ -186,22 +186,12 @@ func StorageMiner(fc config.MinerFeeConfig) func(params StorageMinerParams) (*st
 
 		ctx := helpers.LifecycleCtx(mctx, lc)
 
-		mi, err := api.StateMinerInfo(ctx, maddr, types.EmptyTSK)
+		fps, err := storage.NewWindowedPoStScheduler(api, fc, sealer, sealer, j, maddr)
 		if err != nil {
 			return nil, err
 		}
 
-		worker, err := api.StateAccountKey(ctx, mi.Worker, types.EmptyTSK)
-		if err != nil {
-			return nil, err
-		}
```

Comment:
I assume we just needed to do this to perform the sanity checks inside the miner. I'm now _only_ resolving the worker key there: https://github.com/filecoin-project/lotus/pull/4513/files#diff-e5e6c18c093f7eb4f4d0bb2e19388a831f34ded5ceeff4737091f6dc686aafe6R181-R184